src/frontend/screens/Settings/MapSettings/BackgroundMapInfo.tsx

```diff
@@ -0,0 +1,184 @@
+import { BottomSheetMethods } from "@gorhom/bottom-sheet/lib/typescript/types";
+import MapboxGL from "@react-native-mapbox-gl/maps";
+import * as React from "react";
+import { defineMessages, useIntl } from "react-intl";
+import { ScrollView, StyleSheet, Text, View } from "react-native";
+
+import api from "../../../api";
+import { MAPEO_BLUE, MEDIUM_GREY, WHITE } from "../../../lib/styles";
+import Button from "../../../sharedComponents/Button";
+import Loading from "../../../sharedComponents/Loading";
+import { NativeRootNavigationProps } from "../../../sharedTypes";
+import { DeleteMapBottomSheet } from "./DeleteMapBottomSheet";
+import { useMapStyle } from "../../../hooks/useMapStyle";
+import { convertBytesToMb, DEFAULT_MAP_ID } from "./BackgroundMaps";
+import { DeleteIcon } from "../../../sharedComponents/icons";
+
+const m = defineMessages({
+  removeMap: {
+    id: "screens.Settings.MapSettings.removeMap",
+    defaultMessage: "Remove Map",
+  },
+  cancel: {
+    id: "screens.Settings.MapSettings.cancel",
+    defaultMessage: "Cancel",
+  },
+  subtitle: {
+    id: "screens.Settings.MapSettings.subtitle",
+    defaultMessage:
+      "This map and offline areas attached to it will be deleted. This cannot be undone",
+  },
+  mb: {
+    id: "screens.Settings.MapSettings.mb",
+    defaultMessage: "MB",
+    description: "abbreviation for megabyte",
+  },
+  zoomLevel: {
+    id: "screens.Settings.MapSettings.zoomLevel",
+    defaultMessage: "Zoom Level",
+  },
+  description: {
+    id: "screens.Settings.MapSettings.description",
+    defaultMessage: "Description",
+  },
+  deleteMap: {
+    id: "screens.Settings.MapSettings.deleteMap",
+    defaultMessage: "Delete Map",
+  },
+  useMap: {
+    id: "screens.Settings.MapSettings.useMap",
+    defaultMessage: "Use Map",
+  },
+});
+
+export const BackgroundMapInfo = ({
+  route,
+  navigation,
+}: NativeRootNavigationProps<"BackgroundMapInfo">) => {
+  const { formatMessage: t } = useIntl();
+  const { bytesStored, id, styleUrl, name } = route.params;
+  const [zoomAndDescription, setZoomAndDescription] = React.useState<
+    { zoom?: number; description?: string } | "loading"
+  >("loading");
+
+  const sheetRef = React.useRef<BottomSheetMethods>(null);
+
+  const { setStyleId } = useMapStyle();
+
+  function setStyleAndNavigateHome() {
+    setStyleId(id);
+    navigation.navigate("Home", { screen: "Map" });
+  }
+
+  React.useEffect(() => {
+    api.maps
+      .getStyle(id)
+      .then(style => {
+        // @ts-ignore
+        return api.maps.getTileset(style.sources["raster-source"].url);
```

Comment:
This needs to be improved. I'm just not sure how to do this endpoint.

Reply:
Given what we have to work with, I think I'd appraoch it like this:

1. Fetch the style (you already have this)
2. Although maybe not as applicable in the current state of the map server, but keep in mind that a style can refer to several tilesets. Due to this, I think it would be wise to get information from all of the sources from its `sources` field. Each source is an object as described here: https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/
3. In our case, each source will have a `url` field that points to a saved tileset on the map server, with the format along the lines of `http://127.0.0.1:PORT/tilesets/:tilesetId`. Combined with my other [suggestion](https://github.com/digidem/mapeo-mobile/pull/1029#discussion_r1007144544), you'd have to extract the `:tilesetId`s here which is a little roundabout but should be okay to do using `URL` and some string operations.
4. Once all tileset ids have been extracted, use something like `Promise.all` (or `Promise.allSettled`, depending on desired behavior) to fetch the tilesets from the map server in parallel. Once those have been retrieved, check for the existence of the desired properties of each
  - since a style can refer to several tilesets, not exactly sure what the best case is for the `description` that you're wanting to display e.g. what if a style has 2 tilesets that each have their own description?
  - keep in mind that based on the [TileJSON spec] the `maxzoom` property is optional and if it's not specified, the default is `30`. for the displayed zoom information, guessing you'd take the maximum value that exists among the tilesets that the style references? (basically if any of them don't specify `maxzoom`, then `30` would be the max based on the spec i suppose)
  
  maybe @gmaclennan has additional thoughts on approaching this that may be fewer steps